util/fipstools/acvp/modulewrapper/modulewrapper.cc

```diff
@@ -1086,6 +1098,46 @@ static bool HashMCTSha3(const Span<const uint8_t> args[],
       {Span<const uint8_t>(md[1000])});
 }
 
+static unsigned char* BuildLDTMessage(const bssl::Span<const uint8_t> part_msg, int times) {
+  size_t full_msg_size = part_msg.size() * times;
+  unsigned char* full_msg = (unsigned char*) malloc (full_msg_size);
+  for(int i = 0; i < times; i++) {
+    memcpy(full_msg + i * part_msg.size(), part_msg.data(), part_msg.size());
```

Comment:
```suggestion
    OPENSSL_memcpy(full_msg + i * part_msg.size(), part_msg.data(), part_msg.size());
```
And a few spots down below.

Reply:
Interestingly, the rest of `modulewrapper.cc` uses the regular `memcpy()`, so I used it here as well to follow precedent. I could switch it here and open a new PR to switch the rest to use `OPENSSL_memcpy()` if you think it's worth it though

Reply:
Nah, that would just create more merge conflicts in the future.